opentech/static_src/src/app/src/components/DetailView/index.js

```diff
@@ -26,13 +30,12 @@ class DetailView extends Component {
     }
 
     render() {
-        const { listing, submissionID } = this.props;
-
-        const activeSubmision = !!submissionID;
+        const { listing, isLoading, error, isEmpty, showSubmision } = this.props;
+        const isError = Boolean(error);
```

Comment:
Maybe we want `isError` and `errorMessage` separated

Reply:
👍